VocaDbWeb/Controllers/Api/UserApiController.cs

```diff
@@ -822,5 +822,13 @@ public class PostStatusLimitedModel
 		[Authorize]
 		[AuthenticatedCorsApi(System.Web.Mvc.HttpVerbs.Get)]
 		public ArtistForUserForApiContract GetArtistForUser(int artistId) => GetArtistForUser(_permissionContext.LoggedUserId, artistId);
+
+		[Route("{id:int}/albumCollectionStatuses/{albumId:int}")]
+		public AlbumForUserForApiContract GetAlbumForUser(int id, int albumId) => _queries.GetAlbumForUser(id, albumId);
+
+		[Route("current/albumCollectionStatuses/{albumId:int}")]
+		[Authorize]
+		[AuthenticatedCorsApi(System.Web.Mvc.HttpVerbs.Get)]
+		public AlbumForUserForApiContract GetAlbumForUser(int albumId) => GetAlbumForUser(_permissionContext.LoggedUserId, albumId);

```

Comment:
What should be the URL again?

Reply:
I decided to use kebab-case for newer APIs. `/api/users/{id:int}/followedArtists/{artistId:int}` and `/api/users/current/followedArtists/{artistId:int}` (#725) are camelCase for backward compatibility reasons with other URLs.

Reply:
I decided to use kebab-case for newer APIs. `/api/users/{id:int}/followedArtists/{artistId:int}` and `/api/users/current/followedArtists/{artistId:int}` (#725) are camelCase for backward compatibility reasons with other URLs.